DigitalLearningSolutions.Web/Views/LearningMenu/Completion/Completion.cshtml

```diff
@@ -27,7 +27,7 @@
 
 <dl class="nhsuk-summary-list nhsuk-u-margin-bottom-4">
 
-  @if (Model.DiagnosticAttempts > 0)
+  @if (Model.DiagnosticScore != null && Model.DiagnosticAttempts > 0)

```

Comment:
What about having a `ShowDiagnostic` score field in the model, so that we can test this logic? We could od the same for learning completed for consistency 🙂

Reply:
Yeah I agree. Done in 56b3475 🙂